src/DockerManager.ts

```diff
@@ -42,7 +42,7 @@ export class DockerManager {
         logger.debug(runCommand);
         return util
           .promisify(exec)(runCommand)
-          .then(result => {
+          .then(() => {
             const containerResult: ContainerResult = { pass: true };
             if (outputPath && fs.existsSync(containerOutputPath)) {
```

Comment:
I know this is part of the PR, but this jumped out to me. This conditional and the one starting on line 50 are pretty similar.